application/displayblock.class.inc.php

```diff
@@ -1871,7 +1871,8 @@ public function GetRenderContent(WebPage $oPage, array $aExtraParams, string $sI
 		/** @var array $aToolkitActions Any "legacy" toolkit menu item, which are now displayed in the same menu as the $aRegularActions, after them */
 		$aToolkitActions = [];
 
-		if (!isset($aExtraParams['selection_mode']) || ($aExtraParams['selection_mode'] == "")) {
+		if ((!isset($aExtraParams['selection_mode']) || ($aExtraParams['selection_mode'] == ""))
+			&& (!isset($aExtraParams['menu']) || $aExtraParams['menu'])) {
```

Comment:
```suggestion
		// Display menu actions only if...
		if (
			// ... NOT in a selection mode
			(!isset($aExtraParams['selection_mode']) || ($aExtraParams['selection_mode'] == ""))
			// ... "menu" parameter is NOT EXPLICITLY disabled
			&& (!isset($aExtraParams['menu']) || $aExtraParams['menu'] === "1" || $aExtraParams['menu'] === true)
		) {
```

  * Being explicit about what kind of value we expect in the "menu" param
  * Adding comments to explain in which functional use cases we should display the menu